main.js

```diff
@@ -1,3 +1,36 @@
-import './style.css';
+function addOrRemoveTooltip(){
+    return this.title !== '' ? addTooltip(this) : removeTooltip(this)
+}
+function addTooltip(elem){
+    const title = elem.title;
+    elem.title = '';
+    elem.setAttribute("tooltip", title);
+    const tooltipWrap = document.createElement("div");
+    tooltipWrap.className = 'tooltip';
+    tooltipWrap.appendChild(document.createTextNode(title));
+    const padding = 35;
+    const linkProps = elem.getBoundingClientRect();
+    const tooltipProps = tooltipWrap.getBoundingClientRect();
+    const topPos = linkProps.top - (tooltipProps.height + padding);
+    tooltipWrap.setAttribute('style','top:'+topPos+'px;'+'left:'+linkProps.left+'px;')
```

Comment:
Style is an object of an element
You can replace it like that:
```javascript
tooltipWrap.style.top = topPos + 'px';
tooltipWrap.style.left = linkProps.left + 'px';
```

Or like that:
```javascript
tooltipWrap.style.top = `${topPos}px`;
tooltipWrap.style.left = `${linkProps.left}px`;
```

Reply:
Thanks, I have fixed all